manage.py

```diff
@@ -4,7 +4,12 @@
 
 if __name__ == "__main__":
 
-    settings = 'tola.settings.test' if 'test' in sys.argv else 'tola.settings.local'
+    if 'test' in sys.argv:
+        settings = 'tola.settings.test'
```

Comment:
Can you add here this line?:

`os.environ['DJANGO_SETTINGS_MODULE'] = settings`

If you debug, test settings are not loaded.

Reply:
Done!